plugins/lookup/tss.py

```diff
@@ -82,6 +82,27 @@
                 | items2dict(key_name='slug',
                              value_name='itemValue'))['password']
             }}
+            
+- hosts: localhost
+  vars:
+      secret: >
```

Comment:
There is a lot of magic happening in Jinja2 and Ansible which probably remove it, but generally I think it's better to be explicit and mention "I don't want the trailing newline".
```suggestion
      secret: >-
```